run.py

```diff
@@ -22,6 +22,13 @@ def parse_args(args):
         help="Perform lemmatization (using spaCy)",
         action="store_true",
     )
+    parser.add_argument(
+        "--trainsize",
```

Comment:
--training-set-size

Reply:
can we use --training_set_size? cause hyphen gives error.

Reply:
Yes, that's fine too

Reply:
A question, what error does it give?

Reply:
args.training-set-size considers it three different words training ,set and size .

Reply:
Who considers it three different words? What error do you see exactly?

Reply:
Traceback (most recent call last):
  File "run.py", line 156, in <module>
    main(parse_args(sys.argv[1:]))
  File "run.py", line 87, in main
    model = model_class(args.training-set-size, args.lemmatization)
AttributeError: 'Namespace' object has no attribute 'training'

Reply:
because it contains the word 'set' .

Reply:
OK, so the "who" is Python :P 
The issue is that in Python you can't have variables with dashes in them!
argparse will automatically take care of that, so if you have an argument like "--training-set-size" you can access it as "args.training_set_size". You shouldn't need to change the argument to "--training_set_size" for that to happen.

Reply:
ohh, so python does have some rules :P

Reply:
Done.